x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/QueryRuleCriteria.java

```diff
@@ -23,18 +27,32 @@
 import org.elasticsearch.xcontent.XContentType;
 
 import java.io.IOException;
+import java.util.List;
 import java.util.Locale;
 import java.util.Objects;
 
 import static org.elasticsearch.xcontent.ConstructingObjectParser.constructorArg;
+import static org.elasticsearch.xcontent.ConstructingObjectParser.optionalConstructorArg;
+import static org.elasticsearch.xpack.application.rules.QueryRuleCriteria.CriteriaType.GLOBAL;
 
 public class QueryRuleCriteria implements Writeable, ToXContentObject {
     private final CriteriaType criteriaType;
     private final String criteriaMetadata;
-    private final Object criteriaValue;
+    private final List<Object> criteriaValues;
+
+    private static final Logger logger = LogManager.getLogger(QueryRuleCriteria.class);
 
     public enum CriteriaType {
-        EXACT;
+        GLOBAL,
+        EXACT,
+        EXACT_FUZZY,
```

Comment:
Naming: Exact and fuzzy might be at odds here on meaning...

```suggestion
        FUZZY_MATCH,
```